usaspending_api/database_scripts/broker_matviews/broker_indexes.sql

```diff
@@ -0,0 +1,29 @@
+-- FPDS Indexes
```

Comment:
What is the justification for storing broker indexes in this repo?

Reply:
That should be removed.